aya/src/obj/mod.rs

```diff
@@ -706,32 +738,34 @@ mod tests {
                         value_size: 3,
                         max_entries: 4,
                         map_flags: 5,
-                        id: 0,
```

Comment:
I think you wanted to replace this with `def` too or why bind `def` at all?

Reply:
rustc doesn't like it when `..Default::default()` is called in `Ok()`. 

```
error: expected `}`, found `Default`
   --> aya/src/obj/mod.rs:709:19
    |
703 |             Ok(bpf_map_def {
    |                ----------- while parsing the fields for this pattern
...
709 |                 ..Default::default()
    |                   ^^^^^^^ expected `}`
```

Is there an alternative?

Reply:
yes, switch to `assert_eq` see the comment above

Reply:
Fixed in 86b639b